manage-data/data-store/data-streams/failure-store.md

```diff
@@ -0,0 +1,848 @@
+
+```console
+PUT _index_template/my-index-template
+{
+  "index_patterns": ["my-datastream-*"],
+  "data_stream": { },
+  "template": {
+    "data_stream_options": { <1>
+      "failure_store": {
+        "enabled": true <2>
+      }
+    }
+  }
+}
+```
+
+1. The options for a data stream to be applied at creation time.
+2. The failure store feature will be enabled for new data streams that match this template.
+
+
+After a matching data stream is created, its failure store will be enabled.
+
+### Set up for existing data streams [set-up-failure-store-existing]
+
+Enabling the failure store via [index templates](../templates.md) can only affect data streams that are newly created. Existing data streams that use a template are not affected by changes to the template's `data_stream_options` field.
+To modify an existing data stream's options, use the [put data stream options](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-data-stream-options) API:
+
+```console
+PUT _data_stream/my-datastream-existing/_options
+{
+  "failure_store": {
+    "enabled": true <1>
+  }
+}
+```
+
+1. The failure store option will now be enabled.
+
+
+The failure store redirection can be disabled using this API as well. When the failure store is deactivated, only failed document redirection is halted. Any existing failure data in the data stream will remain until removed by manual deletion or until the data expires due to reaching its max configured retention.
+
+```console
+PUT _data_stream/my-datastream-existing/_options
+{
+  "failure_store": {
+    "enabled": false <1>
+  }
+}
+```
+
+1. Redirecting failed documents into the failure store will now be disabled.
+
+### Enable failure store via cluster setting [set-up-failure-store-cluster-setting]
+
+If you have a large number of existing data streams you may want to enable their failure stores in one place. Instead of updating each of their options individually, set `data_streams.failure_store.enabled` to a list of index patterns in the [cluster settings](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-settings). Any data streams that match one of these patterns will operate with their failure store enabled.
+
+```console
+PUT _cluster/settings
+{
+  "persistent" : {
+    "data_streams.failure_store.enabled" : [ "my-datastream-*", "logs-*" ] <1>
+  }
+}
+```
+1. Indices that match `my-datastream-*` or `logs-*` will redirect failures to the failure store unless explicitly disabled.
+
+Matching data streams will ignore this configuration if the failure store is explicitly enabled or disabled in their [data stream options](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-data-stream-options).
+
+```console
+PUT _cluster/settings
+{
+  "persistent" : {
+    "data_streams.failure_store.enabled" : [ "my-datastream-*", "logs-*" ] <1>
+  }
+}
+```
+```console
```

Comment:
I think these two lines need to be removed to make this one continuous console block, since we use `<2>` below